src/app/containers/LongTermSchedule/LongTermSchedule.tsx

```diff
@@ -20,34 +27,51 @@ interface LongTermScheduleProps extends Partial<ShiftScheduleProps> {
 
 interface LongTermScheduleDispatchProps {
     fetchShifts: () => void;
+    assignShift: (link: { sheriffId: IdType, shiftId: IdType }) => void;
+    unassignShift: (link: { sheriffId: IdType, shiftId: IdType }) => void;
 }
 
 interface LongTermScheduleStateProps {
     shifts: Shift[];
 }
 
 class LongTermSchedule extends React.Component<LongTermScheduleProps
-                                                & LongTermScheduleStateProps
-                                                & LongTermScheduleDispatchProps> {
+    & LongTermScheduleStateProps
+    & LongTermScheduleDispatchProps> {
     componentWillMount() {
         const { fetchShifts } = this.props;
         fetchShifts();
     }
 
     render() {
         const {
-            shifts = []
+            shifts = [],
+            assignShift
         } = this.props;
 
-        const newVisibleTimeStart = moment().startOf('week');
-        const newVisibleTimeEnd = moment().endOf('week');
+        const newVisibleTimeStart = moment().startOf('isoWeek').valueOf();
+        const newVisibleTimeEnd = moment(newVisibleTimeStart).add(5, 'days').valueOf();
```

Comment:
Does this mean that the week is only 5 days long? (i.e. Monday to Friday, Sat and Sun not included)? If so, we may want to confirm this with Jenn.

Reply:
@hollyegrayson I think we can most likely address this when we add in the skipping from week to week.  I just did this to make the long term schedule view a little bit more zoomed in.  But yes we can confirm.  Agree to leave it as for the time being?

Reply:
Sure - lets leave as is for time being